server/src/service-layer/response-models/PaymentResponse.ts

```diff
@@ -14,6 +14,18 @@ export interface MembershipPaymentResponse extends CommonResponse {
   membershipType?: MembershipTypeValues
 }
 
+// Make a data shape matching to the expected response from Stripe API
+export interface MembershipStripeProductResponse extends CommonResponse {
+  data?: {
+    productId: string
+    name: string
```

Comment:
Will have to change name to Membershiptype values - suggest renaming it to Membershiptype